src/ucs/datastruct/mpool.h

```diff
@@ -141,28 +144,46 @@ struct ucs_mpool_ops {
     void         (*obj_str)(ucs_mpool_t *mp, void *obj, ucs_string_buffer_t *strb);
 };
 
+typedef struct ucs_mpool_params {
+    size_t                priv_size;
+    size_t                elem_size;
+    size_t                align_offset;
+    size_t                alignment;
+    unsigned              elems_per_chunk;
+    unsigned              max_chunk_size;
+    unsigned              max_elems;
+    double                grow_factor;
+    const ucs_mpool_ops_t *ops;
+    const char            *name;
```

Comment:
add documentation for each param (to replace from ucs_mpool_init params doc)

Reply:
fixed